src/new-client/src/App/LiveRates/LiveRates.tsx

```diff
@@ -1,11 +1,15 @@
-import { Loader } from "components/Loader"
+import { Loader } from "@/components/Loader"
 import { lazy, Suspense } from "react"
 import { merge } from "rxjs"
-import { currencyPairDependant$ } from "services/currencyPairs"
-import { getHistoricalPrices$, getPrice$ } from "services/prices"
-import styled from "styled-components/macro"
+import {
+  currencyPairDependant$,
+  currencyPairs$,
+} from "@/services/currencyPairs"
+import { getHistoricalPrices$, getPrice$ } from "@/services/prices"
+import styled from "styled-components"
 const LiveRatesCore = lazy(() => import("./LiveRatesCore"))
 
+currencyPairs$.subscribe()
```

Comment:
Was this not necessary before?

Reply:
yes and no... I will explain this better on the stand up. But yeah, before it was working by an accident :see_no_evil: